process/block/preprocess/transactions.go

```diff
@@ -962,7 +961,13 @@ func (txs *transactions) createAndProcessMiniBlocksFromMe(
 		}
 
 		miniBlock.TxHashes = append(miniBlock.TxHashes, txHash)
+		numScCalls := 0
+		if core.IsSmartContractAddress(tx.RcvAddr) {
+			//we need to increment this as to account for the corresponding SCR hash
+			numScCalls++
+		}
 		txs.blockSizeComputation.AddNumTxs(1)
+		txs.blockSizeComputation.AddNumTxs(numScCalls)
```

Comment:
numScCalls here is always 1. call txs.blockSizeComputation.AddNumTxs(1) instead of numSCCalls++. Although this protection should be only for cross shard and per shard.

Reply:
Done